site/content/en/docs/handbook/pushing.md

```diff
@@ -108,9 +142,12 @@ Push docker image to minikube registry:
 docker push $(minikube ip):5000/test-img
 ```
 
-# Building images inside of minikube
+---
 
-Use `minikube ssh` to connect to the virtual machine, and run the `docker build` there:
+## 5. Building images inside of minikube using SSH
+
+Use `minikube ssh` to go inside a minikube node, and run the `docker build` directly there.
```

Comment:
change "to go inside a minikube node" to "to run commands inside the minikube node"